imagelab/DisplayImage.java

```diff
@@ -94,5 +100,17 @@ public DisplayImage(ImgProvider imp, String title, boolean slow) {
         setJMenuBar(myMenuBar);
         setVisible(true);
     }
+    
+    public void remove(){
```

Comment:
I have removed this remove method and will re-commit those changes. This was only used as a preliminary attempt to support the synchronization process but it is understood that this is no longer needed. A call to this method in the ImgProvider.java source has also been removed.